tasks/main.yml

```diff
@@ -3,6 +3,11 @@
 - name: Include OS-specific variables.
   include_vars: "{{ ansible_os_family }}.yml"
 
+# If AmazonLinuxm include needed variables.
+- name: Include OS-specific variables.
```

Comment:
Maybe delete the comment above this line, and set `name: Include variables for Amazon Linux.`